frontend/src/pages/SignIn.js

```diff
@@ -1,84 +1,55 @@
 import React from 'react';
 import './SignIn.global.scss';
-import {
-  Button,
-  Box,
-  Layout,
-  Image,
-  Heading,
-  InputWithLabel,
-  Cell,
-  Row,
-  Col,
-  Container,
-  Card,
-} from 'wix-style-react';
-import { Route } from 'react-router-dom';
+import { Layout } from 'wix-style-react';
+import { useDispatch } from 'react-redux';
+import Cookie from 'js-cookie';
+import { setUser } from '../state/actions';
+import Header from '../components/signInComponents/Header';
+import InputFormContainer from '../components/signInComponents/InputFormContainer';
 
 const SignIn = ({ onLogin }) => {
-  const logo = require('../../assets/logo.jpeg');
-  const handleSignInBtn = history => {
+  const dispatch = useDispatch();
+
+  const handleSignIn = (user, history) => {
+    fetch('https://c5cddfe5.ngrok.io/api/Authentication/login', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(user),
+    })
+      .then(response => {
+        if (response.status === 200)
+          confirmUser(response.json(), history);
+        else if (response.status === 401)
+          alert('Incorrent email or password!');
+        else
+          alert('ERROR: ', response.status);
+      })
+      .catch(err => console.log(err));
```

Comment:
can we also display alert in this situation?

Reply:
No, i deleted it

Reply:
We'll need a loading indicator at some point